query-engine/prisma-models/src/field/mod.rs

```diff
@@ -233,6 +182,32 @@ pub enum DateType {
     DateTime,
 }
 
+impl From<(crate::InternalDataModelRef, walkers::CompositeTypeFieldWalker<'_>)> for Field {
```

Comment:
Are we planning to get rid of the `From` impls at some point?

Reply:
Absolutely. prisma-models is going to get much smaller soon.